Makefile

```diff
@@ -1,16 +1,17 @@
-CORE_TAGS := $(CORE_JDK8_TAG) $(CORE_JDK11_TAG)
-JOBS := $(addprefix core-,${CORE_TAGS})
-CLEAN_JOBS := $(addprefix clean-,${CORE_TAGS})
+CORE_TAG := $(CORE_JDK11_TAG)
+JOB := core-${CORE_TAG}
+CLEAN_JOB := clean-${CORE_TAG}
 
-all: ${JOBS}
-clean: ${CLEAN_JOBS}
-.PHONY: all clean ${JOBS} ${CLEAN_JOBS}
+all: ${JOB}
+clean: ${CLEAN_JOB}
+.PHONY: all clean ${JOB} ${CLEAN_JOB}
 
-${JOBS}: core-%: Dockerfile
-	docker build \
+${JOB}: core-%: Dockerfile
+	docker buildx build \
+	  --platform linux/arm64,linux/amd64 \
```

Comment:
Did we previously not due this due to Java 8 limitations?

Reply:
Pretty much! Or at least that's why I thought of it when making these change. 
Also, I think we just didn't think of it previously, because I don't see us using buildx in our other agents that are just for java8 😅 but this is more reminiscent of what the build actually does so I think it's good to keep in the makefile!